cypress/e2e/forside.cy.ts

```diff
@@ -0,0 +1,30 @@
+describe('Forside tester', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000');
+  });
+  it('Tekst laster fra Sanity', () => {
+    cy.get(`[data-testid='forsideTittel']`).contains('Endringsmelding');
+  });
+  it('Språkvelger endrer språk', () => {
+    cy.get(`[data-testid='språkvelger']`).select('en');
+    cy.get(`[data-testid='forsideTittel']`).contains('Change message');
+  });
+  it('Fornavn hentet fra backend', () => {
+    //Må endre til det fornavnet vi vil hente, mocket
+    cy.get(`[data-testid='hilsenFornavn']`).contains('Hei, Fornavn!');
+  });
+  it('Kan ikke gå videre før samtykkepanel er bekreftet', () => {
+    cy.get(`[data-testid='startKnapp']`).click();
+    //Kan feilmeldingsteksten kanskje hentes på en annen måte eller sjekkes på en annen måte sånn at testen funker selvom teksten blir endret
+    cy.get(`[data-testid='samtykkepanelFeilmelding']`).contains(
+      'Du må samtykke før du kan fortsette',
+    );
```

Comment:
Hmm tror ikke man har tilgang til tekstene her som vi har de andre stedene. Kan hende man kan hente alle tekstene fra sanity før man kjører tekstene for å sjekke at ting stemmer. 

Det kan lages et kort på å utforske dette og så kan vi ha det hardkodet i utgangspunktet. 

PS: Hvis dere ønsker å legge igjen en kommentar om at det kanskje bør gjøre annerledes så legg enten igjen f.eks. `// TODO: Sjekk mot tekst fra Sanity` eller dropp kommentaren og lim dette inn som en kommentar i PR for å få svar fra andre 😁